donkeycar/parts/oled.py

```diff
@@ -0,0 +1,145 @@
+import Adafruit_SSD1306
```

Comment:
Do we need to update the virtual environment to include this dependency?

Reply:
Good catch. I was porting over changes to a different branch. Brought this change over as well.